src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -875,6 +875,7 @@ Object AsyncRPCOperation_sendmany::perform_joinsplit(
             {info.vjsout[0], info.vjsout[1]};
     boost::array<size_t, ZC_NUM_JS_INPUTS> inputMap;
     boost::array<size_t, ZC_NUM_JS_OUTPUTS> outputMap;
+    std::function<int(int)> emptyFunc;
```

Comment:
I wrote the same function but decided not to use it because it was confusing.  If it's an identity gen, why does the function not just return n?  Why n-1?